index.js

```diff
@@ -386,6 +410,122 @@ exports.getLastThreadPost = function (thread) {
   return msg
 }
 
+
+/* post revision utils */
+
+exports.getRevisions = function(ssb, thread, callback) {
+  function collectRevisions(thread, callback) {
+    // this function walks the revisions of a given thread, collecting them up
+    // asyncly
+    var deepThread = pluckRecursively(thread, 'related')[0]
+
+    var threadRevisionLogCB = multicb({pluck: 1})
+    deepThread
+    .filter(function(relatedMsg) { return isaRevisionTo(relatedMsg, thread) })
+    .forEach(function(edit) {
+      exports.createRevisionLog(ssb, edit, threadRevisionLogCB())
```

Comment:
Is this necessary? `relatedMessages` does a recursive thread-collection, but then you're throwing that away and doing your own recursive fetch-from-db here.